pkg/controller/externalippool/validate_test.go

```diff
@@ -432,7 +433,7 @@ func TestParseIPRangeStartEnd(t *testing.T) {
 	for _, testCase := range testCases {
 		t.Run(testCase.name, func(t *testing.T) {
 			// discard parsed net.IP, we only need to make assertions on errMsg.
```

Comment:
comment no longer valid, since `ParseIPRangeStartEnd` is returning 2 `netip.Addr` now